eth-sdk/config.ts

```diff
@@ -1,10 +1,14 @@
-import {defineConfig} from '@dethcrypto/eth-sdk';
+import { defineConfig } from '@dethcrypto/eth-sdk';
 
 export default defineConfig({
   contracts: {
     mainnet: {
       orderJob: '0x656027367B5e27dC21984B546e64dC24dBFaA187',
+      //
       depositManagerJob: '0xa61d82a9127B1c1a34Ce03879A068Af5b786C835',
+      //
+      harvestingJob: '0xEC771dc7Bd0aA67a10b1aF124B9b9a0DC4aF5F9B',
+      usdcSavingsVault: '0x6bAD6A9BcFdA3fd60Da6834aCe5F93B8cFed9598',
```

Comment:
This is just a small comment on what you think is tidier, in the scripts `vault` is instantiated but we only use its address. If the vault doesn't change, maybe it's tidier to add its address to `constants.ts` and just use that instead of creating an instance of the contract.

Anyway, in both cases, if the vault changes, we will need to modify it in either file so just a matter of preference.

Reply:
The first thought when adding the address of the `usdcSavingsVault` here was that I would still use the contract to check the workability of the job, but as it turned out, the contract of the job absolutely satisfies all the needs